.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Changelist
+[Describe or list the changes made in this PR]
+
+### Test Plan
+[Describe how this PR was tested (if applicable)]
+
+### Author/Reviewer Checklist
+- Add an entry to `protocol/CHANGELOG.md` and/or `indexer/CHANGELOG.md` (if applicable).
+- If this PR has changes that modify the way state is read or written, require state migration, or change app hash, manually add the `state-breaking` label.
+- If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal`or `ProcessProposal`, manually add the label `proposal-breaking`.
+- If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
+- If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
+- Manually add any of the following labels: `refactor`, `chore`, `bug`.
```

Comment:
Do we envision that it would be one of these (plus feature) labels?

I would think so and we should encourage ppl to break PRs into logically separate ones if PRs are combinations of these

Reply:
Yeah ideally we have nice granularity of PRs, but generally github PRs tend to be bigger than if we used something like Phabricator because it's tedious (and not common practice) to stack PRs in github. So I definitely think it's forgivable if a PR fits multiple categories. Just add more than one tag.